README.md

```diff
@@ -51,6 +58,8 @@ If you are on a `SNAPSHOT` version, you can force redownloads of modules/depende
  ./gradlew[.bat] clean build --parallel --refresh-dependencies
 ```
 
+Some options are in 'gradle.properties', and you add your own in order to improve gradle speed
```

Comment:
I suppose: if you decide to use the gradle cas overlay then you must be familiar with gradle to some extent. we don't need to teach you as the deployer what maven or gradle those. Best we can do is to refer folks to their official documentation and have them learn it. I think adding a link simply is good, because if and when gradle changes X on their side, we shouldn't carry the maintenance burden to update the same duplicated thing in our docs.